v21.2/cockroach-debug-job-trace.md

```diff
@@ -0,0 +1,74 @@
+---
+title: cockroach debug job-trace
+summary: Learn the commands for collecting debug information from all nodes in your cluster.
+toc: true
+---
+
+{{site.data.alerts.callout_danger}}
+We strongly recommend only using `cockroach debug job-trace` when working directly with the Cockroach Labs support team.
```

Comment:
'support team' could link to our support resources page: https://www.cockroachlabs.com/docs/v21.1/support-resources.html